_format/1.1/index.md

```diff
@@ -190,8 +192,18 @@ Here's how an article (i.e. a resource of type "articles") might appear in a doc
 
 #### <a href="#document-resource-object-identification" id="document-resource-object-identification" class="headerlink"></a> Identification
 
-Every [resource object][resource objects] **MUST** contain an `id` member and a `type` member.
-The values of the `id` and `type` members **MUST** be strings.
+As noted above, every [resource object][resource objects] **MUST** contain a
+`type` member. Every resource object **MUST** also contain an `id` member,
+except when the resource object originates at the client and represents a new
```

Comment:
The wording here suggests that it MUST contain an `id` except when creating, then it MUST NOT contain an `id`. Which conflicts with http://jsonapi.org/format/#crud-creating-client-ids.

Clarifying that it MAY have an `id` when creating a new resource would remove this apparent contradiction.

Reply:
When I'm reading this my interpretation is not that it says that it "*MUST NOT contain an id*". I.e. I think the text is fine as it is (I don't see the conflict).

But perhaps I misunderstand your comment, @wimleers?

Reply:
You didn't misunderstand my comment. I _do_ think that this can be read as to-be-created resource objects are not allowed to contain an `id` member. IMHO the point of a spec is to remove as much ambiguity as possible, which is why I suggested making this more explicit.